server/mysterium_api_fake.go

```diff
@@ -55,6 +61,7 @@ func (client *ClientFake) FindProposals(nodeKey string) (proposals []dto_discove
 	return proposals, nil
 }
 
+// SendSessionStats heartbeats that session is still active
```

Comment:
This should also include that stats are being sent here about upload and download amounts.

Reply:
Improved